tests/test_rubicon.py

```diff
@@ -238,6 +238,49 @@ def test_polymorphic_static_method(self):
         with self.assertRaises(ValueError):
             Example.tripler(1.234)
 
+    def test_pass_int_array(self):
+        """A list of Python ints can be passed as a Java int array."""
+        Example = JavaClass("org/beeware/rubicon/test/Example")
+        self.assertEqual(3, Example.sum_all_ints([1, 2]))
+
+    def test_heterogenous_list(self):
+        """A list of mixed types raise an exception when trying to find the right Java method."""
+        Example = JavaClass("org/beeware/rubicon/test/Example")
+        with self.assertRaises(ValueError):
+            Example.sum_all_ints([1, "two"])
+        with self.assertRaises(ValueError):
+            Example.sum_all_floats([1.0, "two"])
+        with self.assertRaises(ValueError):
+            Example.sum_all_doubles([1.0, "two"])
```

Comment:
Missing test case here is 'first argument == "two"'

Reply:
Added